models/classes/dto/OldProperty.php

```diff
@@ -46,4 +48,11 @@ public function getPropertyType(): ?core_kernel_classes_Resource
     {
         return $this->propertyType;
     }
+    /**
+     * @return string|null
+     */
+    public function getrangeUri(): ?string
```

Comment:
- No need for anotation
- Use camelCase to separate words, please :) 

```suggestion
    public function getRangeUri(): ?string
```